packages/psss/src/api/queries/useActiveAlerts.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { MIN_DATE, SYNDROMES } from '../../constants';
+import { getPeriodByDate } from '../../utils';
+import { useReport } from './helpers';
+
+export const useActiveAlerts = (period, orgUnitCodes) => {
```

Comment:
This is looking tight! The meshing of the server response with front end display data (syndomeName) is really clean .